### PR TITLE
KOA-5578 Update fab to semantic tokens

### PR DESCRIPTION
**Pull request by marianeum**
The disabled colour will need to be updated when the button disabled colours are available

+ [x] Check this if you have read and followed the [contributing guidelines](https://github.com/Skyscanner/backpack-android/blob/main/CONTRIBUTING.md)

Remember to include the following changes:
+ [x] Component `README.md`
+ [x] Tests
+ [x] Docs (either update [backpack-docs](https://github.com/Skyscanner/backpack-docs) now, or create a follow up ticket)

If you are curious about how we review, please read through the [code review guidelines](https://github.com/Skyscanner/backpack/blob/main/CODE_REVIEW_GUIDELINES.md)

### CODE REVIEW

File: Backpack/src/main/java/net/skyscanner/backpack/fab/BpkFab.kt [hunk 1]
```diff
@@ -59,14 +59,16 @@ open class BpkFab @JvmOverloads constructor(
...
+      context.getColor(R.color.bpkTextDisabled)
+    )
     this.isClickable = isEnabled
     this.stateListAnimator = AnimatorInflater.loadStateListAnimator(context, R.drawable.bpk_button_state_animator)
     this.elevation = resources.getDimensionPixelSize(R.dimen.bpkElevationBase).toFloat()
     this.backgroundTintList = getColorSelector(
       backgroundColour,
-      backgroundColour,
-      context.getColor(R.color.bpkSkyGrayTint06)
+      context.getColor(R.color.__buttonDisabledBackground)
```
Comment (marianeum) [inline Backpack/src/main/java/net/skyscanner/backpack/fab/BpkFab.kt:71]: once the new button tokens are available we can use the new colours here too

File: app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt [hunk 2]
```diff
@@ -298,14 +298,14 @@ object ComponentRegistry {
         TAB_TITLE_COMPOSE composeStory { FlareStory() },
       ),
     ),
+    "Floating Action Button" story NodeData { Story of R.layout.fragment_fab },
```
Comment (marianeum) [inline app/src/main/java/net/skyscanner/backpack/demo/data/ComponentRegistry.kt:301]: this was in the wrong position alphabetically

**Review by olliecurtis — APPROVED** [review]
LGTM! One concern is are teams going to be aware this big change of colour to the FAB before we release this as looking at the screenshots this is a significant change colour wise from green to blue?

**Comment by marianeum** [comment]
Yep - I had a look at current FAB usages and it looks like almost all of them set a custom background colour - when making the update I'll also apply a custom foreground colour, as we now switch between day/night mode where this wasn't done yet.

**Pull request merged**